test/opexecuter.cpp

```diff
@@ -129,9 +129,9 @@ OpExecuter::OpExecuter(const Scope sc, const string test_op,
 // Destructor
 OpExecuter::~OpExecuter() {}
 
-void OpExecuter::RunTest() {
+void OpExecuter::RunTest(string ng_backend_name) {
```

Comment:
You could use `const string&` instead of `string`

Reply:
@avijit-nervana Incorporated the change.